### PR TITLE
Added an example with rollup

### PR DESCRIPTION
**Pull request by mattpocock**

### CODE REVIEW

File: examples/with-rollup/.gitignore [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+coverage
+
+# next.js
+.next/
+out/
+build
+
+# misc
+.DS_Store
+*.pem
```
Comment (mehulkar) [inline examples/with-rollup/.gitignore:18]: Is this a copy paste or are there actually any docs or workflows that generate certs somewhere?

File: examples/with-rollup/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+This Turborepo includes the following packages/apps:
+
+### Apps and Packages
+
+- `ui`: a React component library used by the `web` application, compiled with Rollup
+- `web`: a [Next.js](https://nextjs.org) app
+- `eslint-config-custom`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
+- `tsconfig`: `tsconfig.json`s used throughout the monorepo
+
+Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
```
Comment (mehulkar) [inline examples/with-rollup/README.md:16]: "workspace" ?

File: examples/with-rollup/turbo.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**"]
```
Comment (mehulkar) [inline examples/with-rollup/turbo.json:5]: do we need to add `outputs`  for `ui` package?  that would help illustrate how turbo works across multiple workspaces

File: examples/with-rollup/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+# Turborepo starter with Rollup
```
Comment (mehulkar) [inline examples/with-rollup/README.md:1]: This should include something about how to use this folder, as people are expecting to come to examples and use it as a template. Similar to the design-system example: https://github.com/vercel/turborepo/blob/main/examples/design-system/README.md#getting-started.

**Review by mehulkar — APPROVED** [review]
I didn't clone down and try running it, but code looks good! General question about how many examples we want to add (and maintain), but we can discuss that off thread.

File: examples/with-rollup/.gitignore (hunk 1 above)
Comment (anthonyshew) [reply examples/with-rollup/.gitignore:18]: This is a typical copy paste in a JS/TS .gitignore. At least I'm used to seeing it anyway. 😄

File: examples/with-rollup/README.md (hunk 2 above)
Comment (anthonyshew) [reply examples/with-rollup/README.md:16]: I like it but will stay consistent with our other examples for now.

File: examples/with-rollup/turbo.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "$schema": "https://turborepo.org/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**", "build/**", ".next/**"]
+    },
+    "lint": {
+      "outputs": []
+    },
+    "dev": {
+      "cache": false
```
Comment (mehulkar) [inline examples/with-rollup/turbo.json:11]: persistent:true

**Review by mehulkar — APPROVED** [review]

**Pull request merged**